client/package.json

```diff
@@ -75,6 +75,7 @@
     "jquery": "2.2.4",
     "jquery-ui": "^1.12.1",
     "json-loader": "^0.5.4",
+    "lodash": "^4.17.4",
```

Comment:
We have discussed this before, we'd prefer to not use lodash if only a few places benefits from it. There're costs in extra bundle size and learning curve.  Would appreciate if you can implement the notification without it.